### PR TITLE
Make tokenize tests readable

### PR DESCRIPTION
**Pull request by krammnic**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [ ] update tests and/or documentation
- [X] clean up

Please link to any issues this PR addresses.

#### Changelog
What are the changes made in this PR?
* #1823 

#### Test plan
Please make sure to do each of the following if applicable to your PR. If you're unsure about any one of these just ask and we will happily help. We also have a [contributing page](https://github.com/pytorch/torchtune/blob/main/CONTRIBUTING.md) for some guidance on contributing.

- [ ] run pre-commit hooks and linters (make sure you've first installed via `pre-commit install`)
- [ ] add [unit tests](https://github.com/pytorch/torchtune/tree/main/tests/torchtune) for any new functionality
- [ ] update [docstrings](https://github.com/pytorch/torchtune/tree/main/docs/source) for any new or updated methods or classes
- [X] run unit tests via `pytest tests`
- [ ] run recipe tests via `pytest tests -m integration_test`
- [ ] manually run any new or modified recipes with sufficient proof of correctness
- [ ] include relevant commands and any other artifacts in this summary (pastes of loss curves, eval results, etc.)

#### UX
If your function changed a public API, please add a dummy example of what the user experience will look like when calling it.
Here is a [docstring example](https://github.com/pytorch/torchtune/blob/6a7951f1cdd0b56a9746ef5935106989415f50e3/torchtune/modules/vision_transformer.py#L285)
and a [tutorial example](https://pytorch.org/torchtune/main/tutorials/qat_finetune.html#applying-qat-to-llama3-models)

- [X] I did not change any public API
- [ ] I have added an example to docs or docstrings

Will require changes in CI(pre-commit run makes expected_tokens lists unreadable)

### CODE REVIEW

**Comment by krammnic** [comment]
cc: @RdoubleA @joecummings What do you think? With current lint formatting working with this tests is really awful. Pretty minor fix

**Comment by krammnic** [comment]
Lint CI at this point should be changed, is not the formating will be really bad

File: tests/torchtune/models/gemma/test_gemma_tokenizer.py [hunk 1]
```diff
@@ -17,454 +17,62 @@ def tokenizer(self):
...
+                182, 9, 222, 1661, 191, 332, 92, 92, 24, 24, 4, 2]  # noqa
+
+    def test_tokenize_messages(self, tokenizer, expected_tokens):
         messages = [
             Message(
                 role="user",
                 content="Below is an instruction that describes a task. Write a response "
-                "that appropriately completes the request.\n\n### Instruction:\nGenerate "
-                "a realistic dating profile bio.\n\n### Response:\n",
+                        "that appropriately completes the request.\n\n### Instruction:\nGenerate "
```
Comment (joecummings) [inline tests/torchtune/models/gemma/test_gemma_tokenizer.py:37]: What are these changes?

File: tests/torchtune/models/llama3/test_llama3_tokenizer.py [hunk 2]
```diff
@@ -311,21 +147,17 @@ def test_tokenizer_vocab_size(self, tokenizer):
         assert tokenizer.vocab_size == 128257
 
     def test_tokenize_text_messages(
-        self, tokenizer, user_text_message, assistant_text_message
+            self, tokenizer, user_text_message, assistant_text_message
```
Comment (joecummings) [inline tests/torchtune/models/llama3/test_llama3_tokenizer.py:150]: ?

**Review by joecummings — COMMENTED** [review]
Looks good -  I will look into the linter issue. My naive assumption was that `noqa` should work. 

Couple questions about possibly unintended formatting errors

File: tests/torchtune/models/gemma/test_gemma_tokenizer.py (hunk 1 above)
Comment (krammnic) [reply tests/torchtune/models/gemma/test_gemma_tokenizer.py:37]: Or, something change from my local linter changes probably, will fix

File: tests/torchtune/models/llama3/test_llama3_tokenizer.py (hunk 2 above)
Comment (krammnic) [reply tests/torchtune/models/llama3/test_llama3_tokenizer.py:150]: Same is here

**Comment by krammnic** [comment]
I assume that fixed

**Comment by joecummings** [comment]
> I assume that fixed

Grr still failing. Mind if I take a look?

**Comment by krammnic** [comment]
Isn't it about lines with # noqa?

**Comment by krammnic** [comment]
Ah, I see:
`tests/torchtune/models/llama3/test_llama3_tokenizer.py:226:183: B950 line too long (182 > 120 characters)`

**Comment by krammnic** [comment]
Fixed

**Comment by krammnic** [comment]
One more...

**Comment by krammnic** [comment]
fixed with flake

**Comment by krammnic** [comment]
@joecummings Sorry for such lint failures, but I could not able to run pre-commit run --all-files due to current fixes

**Comment by krammnic** [comment]
@joecummings Probably I found solution we need to use both # noqa and # fmt: skip. But I really don't like it

**Comment by krammnic** [comment]
Oh, I broke something...

**Comment by krammnic** [comment]
I don't know what is it, this tests are passing on my local and branch is up to date

**Comment by krammnic** [comment]
Isn't it related too #1886? Some weird fail with torchao

**Comment by krammnic** [comment]
Can we restart CI here? Or I'm not sure how to fix some torchao unrelated stuff

**Comment by krammnic** [comment]
@felipemello1 @RdoubleA Maybe you can comment how to fix this torchao thing? Really strange and probably just CI rerun can't help

**Comment by RdoubleA** [comment]
@krammnic have you merged from main after this PR was checked in? https://github.com/pytorch/torchtune/pull/1886

**Comment by krammnic** [comment]
> @krammnic have you merged from main after this PR was checked in? #1886

Oh, I see. Let me merge it, yes.

**Comment by krammnic** [comment]
Fixed

**Comment by krammnic** [comment]
Can someone restart CI?

**Comment by krammnic** [comment]
resolved

**Comment by krammnic** [comment]
@RdoubleA Can we fix Qwen2 and Qwen2.5 tests in separate PR? I will open it immediately after we merge this without other models

**Comment by RdoubleA** [comment]
@krammnic Sorry for the delay, that sounds good to me. Looks like we just need to resolve merge conflicts and we can land this.

**Comment by RdoubleA** [comment]
@krammnic Went ahead and did the merge with main, thanks again for your help!

**Pull request merged**